### PR TITLE
Exercise 4

### PR DESCRIPTION
**Pull request by Ulaize**
Done through reverse engineering.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -75,6 +39,21 @@
         
         <section class="schedule">
             <h2>Schedule</h2>
+            <table style="width:100%">
```
Comment (nyanofthemoon) [inline index.html]: @Ulaize Use of inline styles, unless absolutely impossible to do so otherwise, is very very bad practice. Please change.

Comment (Ulaize) [inline index.html]: I fixed it in the new commits @nyanofthemoon Thank you.

File: css/main.css [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    border-collapse: collapse;
+    border-spacing: 0;
+}
+table th {
+    padding: 6px 13px;
+    border: 1px solid #000;
+}
+table td {
+    padding: 6px 13px;
+    border: 1px solid #000;
```
Comment (nyanofthemoon) [inline css/main.css]: @Ulaize Almost there! Please update the table declarations from border color #000 to #DDD

Comment (Ulaize) [inline css/main.css]: This comment seems to have disappeared. I actually changed the color cos it
looked bad to me haha. Changes have been made and you can see them here:

901ff55e8fd1019a41869dffc740f2a5edc6a8ef


*Ulaize Hernández Troyas*
www.creative-tweed.com
www.geotweed.com

<http://www.twitter.com/ulaize>
<http://ie.linkedin.com/in/ulaizehernandez/>
*Information in this message is confidential. It is intended solely for the
person to whom it is addressed.*

Comment (Ulaize) [inline css/main.css]: OK. I found it @nyanofthemoon

**Pull request closed without merging**